Gemfile

```diff
@@ -14,3 +14,8 @@ gem "rubocop-rails-omakase", require: false
 
 # Start debugger with binding.b [https://github.com/ruby/debug]
 # gem "debug", ">= 1.0.0"
+
+group :test do
+  gem "actioncable"
+  gem "turbo-rails"
```

Comment:
@ZachGrande why add turbo-rails here? This gem takes pains to avoid a turbo-rails dependency. Part of the reason for avoidance of turbo-rails is that it modifies global state in unexpected ways. I realize this is just being added in test env, but that also may have unintended consequences for the tests which may confuse things.

Reply:
Ahh...good point. Ok...let's keep it to move this forward. There's probably a better level of abstraction here...perhaps an adapter pattern like ActiveRecord has adapters for different databases...but I don't know what that looks like yet, so yea let's just proceed as is. Thanks!